### PR TITLE
Adapt Reviews module to TopBunk

### PR DESCRIPTION
**Pull request by dweinzimmer**
Ticket: https://trello.com/c/IXPookbG/13-modify-legacy-application-to-be-topbunk-reviews-and-run-locally-on-my-machine 

This pull request adapts the legacy source code to (1) work on my machine, and (2) represent the TopBunk application and its specifications, including using the IP address range reserved for me. I have also made changes to the `package.json` scripts to automate loading of data for future users/contributors, and have made updates to the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,34 @@
...
+
+## Related Projects
+
+  - https://github.com/TopBunkNYC/Description
+  - https://github.com/TopBunkNYC/Booking
+  - https://github.com/TopBunkNYC/Neighborhood 
+
+## Development
+To run the application in non-development mode from within the root directory: 
+```sh
```
Comment (wyatt-troia) [inline README.md]: What is ```sh ``` ?

Comment (wyatt-troia) [reply README.md]: with backticks

File: package.json [hunk 2]
```diff
@@ -8,7 +8,8 @@
     "react-dev": "webpack -d --watch",
     "build": "webpack -p",
     "start": "node server/index.js",
-    "seed": "mysql -u root -p --host=staybnb.c1bo9omfdnqb.us-east-2.rds.amazonaws.com reviews",
+    "seed": "mysql -u root -p < database/schema.sql",
+    "postseed": "mysql -u root -p < database/data.sql",
```
Comment (wyatt-troia) [inline package.json]: What is the purpose of postseed?

**Comment by wyatt-troia** [comment]
Overall this looks good. I had some questions about the README and the postseed script, above.

File: package.json (hunk 2 above)
Comment (dweinzimmer) [reply package.json]: For any script name, a script of the same name with `pre` prepended will be run before it, and one with `post` prepended will be run after it. So `postseed` is automatically executed after `seed` when I run `npm run seed` on the command line. [Here's an article about it.](http://www.marcusoft.net/2015/08/pre-and-post-hooks-for-npm-scripting.html)

It is basically the same as if I had written `"mysql -u root -p < database/schema.sql && mysql -u root -p < database/data.sql"`. I've tested that running `npm run seed` does successfully create and populate my database tables.

File: README.md (hunk 1 above)
Comment (dweinzimmer) [reply README.md]: For Markdown syntax, the three backticks signify the beginning of a code snippet, and if you insert the corresponding language abbreviation Github will perform syntax highlighting appropriate for that language. `sh` triggers `bash` syntax highlighting (and `js` would trigger JavaScript syntax highlighting).

**Comment by dweinzimmer** [comment]
@wyatt-troia thanks for the comments and taking a thorough look at the code! See my responses above. I'll merge the pull request now.

**Pull request merged**